docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -731,7 +731,7 @@ to slow the test down.
 testcafe chrome my-tests --speed 0.1
 ```
 
-If the speed is also specified for an individual action, the action's speed setting overrides the test speed.
+The action's speed setting overrides the test speed specified for an individual action.
```

Comment:
This doesn't make sense.

`The action's speed setting` and `the test speed specified for an individual action` are the same thing.

You might have misread the original sentence (or the sentence was too vague) 😄

Reply:
> If you specify speed for individual action, this setting overrides the global test speed.